frontend/views/pages/GroupChat.vue

```diff
@@ -1,10 +1,54 @@
 <template lang='pug'>
 page(pageTestName='dashboard' pageTestHeaderName='groupName')
-  template(#title='') {{ L('Group Chat') }}
+  template(#title='')
+    .c-header
+      i(
+        v-if='summary.private !== undefined'
+        :class='`icon-${ summary.private ? "lock" : "hashtag" } c-group-i`'
+      )
+      | {{summary.title}}
+      menu-parent
+        menu-trigger.c-menu-trigger.is-icon-small
+          i.icon-angle-down.c-menu-i
+
+        menu-content
+          menu-header
+            i18n Channel Options
+
+          ul
+            menu-item(@click='openModal("EditChannelNameModal")')
+              i18n Rename
+            menu-item(@click='openModal("GroupMembersAllModal", {name: summary.title})')
+              i18n Members
+            menu-item.c-separator(tag='router-link' itemid='hash-3' to='/group-settings')
+              i18n Notifications settings
+            menu-item(@click='openModal("LeaveChannelModal")')
+              i18n(:args='{ groupName: summary.title }') Leave {groupName}
+            menu-item.has-text-danger(@click='openModal("DeleteChannelModal")')
+              i18n Delete channel
+
+  template(#description='')
+    .c-header-description
+      button.is-unstyled.c-link(
+        @click='openModal("GroupMembersAllModal")'
+      ) {{ members.size + ' ' + L('members') }}
```

Comment:
A few issues with this localization:

- In some languages the number of members might come after the word "members"
- By not including the variable in the string-to-be-translated, we deprive the translator of the context they need to translate this sentence

So this string must include the variable for the number of members within the string-to-be-translated, and whether that's done here or in a computed property or via an `i18n` tag with `:args` doesn't matter (although I don't remember if those support the `button` tag...).

Either way, the string-to-be-translated should look something like: `'{numMembers} members'`. Double-check if similar issues exist anywhere else, I might've missed them.